src/services/WithdrawalService.ts

```diff
@@ -102,4 +111,18 @@ export default class WithdrawalService {
             return { error };
         }
     }
+
+    static async getWithdrawals(address: string, member: string, rewardId: number, state: number) {
+        try {
+            const withdrawals = await Withdrawal.find({
+                ...member && !member.startsWith('0x000') ?  { beneficiary: member } : {},
```

Comment:
@JustinaMary Why are you checking for 0x000 here? That makes no sense.

Reply:
> If I put the check to validate the address we won't be able to pass 0x00000000000000000000000000000000000000 in the parameter

Then you should validate for that address, since in theory its possible that a valid wallet address starts with 0x000. Next to that this kind of validation should happen in the validation middleware, not in the controller or service layer.